packages/adapter-react/src/index.tsx

```diff
@@ -50,17 +49,16 @@ export const initI18nReact = <
 	const context = React.createContext({} as I18nContextType<L, T, TF>)
 
 	const component: React.FunctionComponent<TypesafeI18nProps<L>> = (props) => {
-		const [locale, _setLocale] = React.useState<L>(null as unknown as L)
-		const [LL, setLL] = React.useState<TF>(getFallbackProxy<TF>())
```

Comment:
What happens when you don't pass the locale to the context? I think the fallback is still needed

Reply:
I returned getFallbackProxy in this case, although locale shouldn't be a undefined according to type:

https://github.com/ivanhofer/typesafe-i18n/blob/9ff0e7c58db909f1a8e4f08b7d145a93fe903fd2/packages/adapter-react/src/index.tsx#L23-L26